vector/src/main/java/im/vector/app/features/spaces/create/CreateSpaceViewModel.kt

```diff
@@ -204,12 +235,31 @@ class CreateSpaceViewModel @AssistedInject constructor(
                 }
                 _viewEvents.post(CreateSpaceEvents.NavigateToChoosePrivateType)
             } else {
+                // it'a public space, let's check alias
+                val aliasLocalPart = if (state.aliasManuallyModified) state.aliasLocalPart else getAliasFromName(state.name)
```

Comment:
`aliasLocalPart` should always be up to date, whatever the value of `aliasManuallyModified` is I think. Why do you call again `getAliasFromName()`?

Reply:
It is not always up to date as per current code.

Reply:
OK